proxy/config.ini

```diff
@@ -12,3 +12,5 @@ pushLogLevel=SUMMARY
 pushValidationLevel=NUMERIC_ONLY
 
 idFile=wavefront_test_id
+
+traceDerivedCustomTagKeys=span.kind
```

Comment:
Users will have different config files, so instead of adding it here, let's programmatically include `span.kind` when handling `traceDerivedCustomTagKeys` in the code.